bricks/basic-bricks/src/app-bar/index.tsx

```diff
@@ -11,6 +11,11 @@ class AppBarElement extends UpdatingElement {
   @property()
   documentId: string;
 
+  @property({
+    attribute: false,
+  })
+  private noCurrentApp: boolean;
```

Comment:
`type: Boolean/String/Number` 的简单属性可以有原生的 attribute 对应，尽量有限使用它；

另外属性是 public 的。

```suggestion
  @property({
    type: boolean,
  })
  noCurrentApp: boolean;
```